pkg/otelcollector/config/builtin_config.go

```diff
@@ -51,6 +51,9 @@ func AddPrometheusRemoteWriteExporter(config *OTELConfig, promClient promapi.Cli
 	// tags, so they are not properly read by OTEL. Need to use bare maps instead.
 	config.AddExporter(otelconsts.ExporterPrometheusRemoteWrite, map[string]any{
 		"endpoint": endpoint.String(),
+		"resource_to_telemetry_conversion": map[string]any{
```

Comment:
Authors are planning to deprecate this option so we should be careful. Just saying.

Reply:
Created issue to track this https://github.com/fluxninja/aperture/issues/1228